fmralign/metrics.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+from scipy.stats import pearsonr
+from sklearn.metrics import r2_score
+
+
+def score_table(ground_truth, prediction, masker, loss,
+                multioutput='raw_values'):
```

Comment:
'multi_output' is more pythonic isn't it ?

Reply:
multioutput for consistency with api of sklearn r2_score (and metrics in general)